brayns/engine/Engine.h

```diff
@@ -145,6 +145,13 @@ class Engine
     void addRenderer(const std::string& name,
                      const PropertyMap& properties = {});
 
+    /**
+     * Add a new camera with optional properties. The camera registration for a
+     * concrete engine is specific to the actual engine, e.g.
+     * OSP_REGISTER_CAMERA for OSPRay.
+     */
+    void addCamera(const std::string& name, const PropertyMap& properties = {});
```

Comment:
Both this name and addRenderer can be confusing. You are not adding a camera, you are adding a camera type. At least the documentation should make it clear. Otherwise somebody could think that can use this method to render a scene from multiple viewpoints.

Reply:
You are the one who would think that :stuck_out_tongue: I can rename it to `addCameraType`.